adapters/cointraffic/cointraffic.go

```diff
@@ -0,0 +1,128 @@
+package cointraffic
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"net/http"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+// Builder builds a new instance of the Foo adapter for the given bidder with the given config.
```

Comment:
Thanks for using the template from the docs. Please rename "Foo" to "Cointraffic" for your adapter.